content/actions/reference/workflow-commands-for-github-actions.md

```diff
@@ -262,7 +262,19 @@ steps:
 
 ### Setting an environment variable
 
-`echo "{name}={value}" >> $GITHUB_ENV`
```

Comment:
I think this is a great addition! I would recommend replacing the snippet in the  "Example" section. That way, users can see the syntax before seeing an example. 

So lines 281-287 could instead be something like:

#### Example

{% raw %}
```bash
    steps:
    - name: Set the value
      run: |
          echo "action_state=yellow" >> $GITHUB_ENV
    - name: Use the value
      run: |
          echo $action_state
```
{% endraw %}


What do you think?

Reply:
Hey @skedwards88 actually that's a good idea, I completely missed the example bit. Changing that now :)